.travis.yml

```diff
@@ -15,10 +15,3 @@ before_install:
 script:
 - bundle exec bin/rails app:test:providers:amazon:setup app:test:providers:amazon
 after_script: bundle exec codeclimate-test-reporter
-notifications:
-  webhooks:
-    urls:
-    - https://webhooks.gitter.im/e/7af45001fe6b7a4039f2
-    on_success: change
-    on_failure: always
-    on_start: never
```

Comment:
This part should not be removed.  This will *always* be different repo-to-repo, or rather the line of the URL will always be different.